README.md

```diff
@@ -4,7 +4,7 @@
 **Requires at least:** 4.4  
 **Requires PHP:** 5.6.20  
 **Tested up to:** 5.7  
-**Stable tag:** 1.6.6  
+**Stable tag:** 1.6.7-beta1  
```

Comment:
We shouldn't be updating the stable tag in a commit for a bug fix.

Reply:
Newest commit removed all beta tags